src/app/routes/liveRadio/getInitialData/index.js

```diff
@@ -19,11 +19,14 @@ const getPageIdentifier = path([
 const getHeading = path(['content', 'blocks', 0, 'text']);
 const getBodySummary = path(['content', 'blocks', 1, 'text']);
 
-export default async ({ path: pathname }) => {
+export default async ({ path: pathname, pageType }) => {
   try {
     const liveRadioDataPath = overrideRendererOnTest(pathname);
-    const { json, status } = await fetchPageData(liveRadioDataPath);
-    const pageType = { metadata: { type: 'Live Radio' } };
+    const { json, status } = await fetchPageData({
+      path: liveRadioDataPath,
+      pageType,
+    });
+    const liveRadioPageType = { metadata: { type: 'Live Radio' } };
```

Comment:
Double declaration of variables with pageType in their name is a bit confusing here, is it worth just passing the metadata object straight into `pageData`?

Reply:
to make things more confusing there is a page type here too https://github.com/bbc/simorgh/blob/latest/src/app/routes/liveRadio/index.js#L12

Reply:
Yeah, it's confusing. It's not ideal that page type is set to `media` instead of `liveRadio` (because this is used in logging to help us distinguish between the different media page types).

Reply:
I have passed the `metadata` object straight into `pageData`, instead of having a separate confusing variable.

I have also started a draft PR to sort out the discrepancy in page types for the media pages (excluding MAPs) - I think I might redo this on a page type by page type basis though, so fewer files are affected. https://github.com/bbc/simorgh/pull/7042